test/api/v3/integration/user/DELETE-user.test.js

```diff
@@ -30,6 +30,20 @@ describe('DELETE /user', () => {
     });
   });
 
+  it('does not need a password if logged in through Facebook', async () => {
+    await user.auth.facebook.id.del('/user', {
```

Comment:
Yea, you will have to alter the generator (which uses the Mongoose Model directly, I believe). You will not be able to update the id via the user update route.